scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -317,6 +317,26 @@ psa_status_t psa_driver_wrapper_sign_hash(
             if( status != PSA_ERROR_NOT_SUPPORTED )
                 return( status );
 #endif /* PSA_CRYPTO_DRIVER_TEST */
+#if defined (MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED)
+            if( PSA_KEY_TYPE_IS_ECC( attributes->core.type ) &&
+                PSA_ALG_IS_ECDSA(alg) &&
+                !PSA_ALG_ECDSA_IS_DETERMINISTIC( alg ) &&
+                PSA_KEY_TYPE_ECC_GET_FAMILY(attributes->core.type) == PSA_ECC_FAMILY_SECP_R1 &&
+                attributes->core.bits == 256 )
+            {
+                status = p256_transparent_sign_hash( attributes,
+                                                      key_buffer,
+                                                      key_buffer_size,
+                                                      alg,
+                                                      hash,
+                                                      hash_length,
+                                                      signature,
+                                                      signature_size,
+                                                      signature_length );
+                if( status != PSA_ERROR_NOT_SUPPORTED )
```

Comment:
Some inconsistent formatting in this file. Please run the style-checker/fixer.

Reply:
As far as I know, the code style script doesn't work on `jinja` templates. This whole file uses the old code style. I've pushed a change that fixes some incorrect alignments of function arguments however. I think migrating this file to the new code style should be handled by a separate PR. Alternatively, if it's not urgent, we can wait for the code style script to be updated.

Reply:
The code style script does not and will not work on Jinja template. It's a tool limitation: the syntax must be close enough to C. The best we could do is to verify the generated code, but there would be no `--fix` mode.